java-symbol-solver-core/src/test/java/com/github/javaparser/symbolsolver/model/typesystem/TypeVariableUsageTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Federico Tomassetti
```

Comment:
Probably we should change this line to "JavaSymbolSolver team" or something like this